frontend/public/style.scss

```diff
@@ -10,6 +10,7 @@
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/variables";
 @import "~bootstrap-sass/assets/stylesheets/bootstrap/mixins";
 @import '~patternfly-react-extensions/dist/sass/patternfly-react-extensions';
+@import '~patternfly-react/dist/sass/patternfly-react';
```

Comment:
Does webpack only include the relevant SCSS?  Or does this import all of it?

Reply:
I am not sure actually, maybe @jeff-phillips-18 can help answer?

Reply:
Yeah, this will pull all of the patterfly-react sass files, though it's not _that_ much.
Should probably just 
```
@import '~patternfly-react/dist/sass/modeless-overlay';
```
for now.

Reply:
And I believe this import (and the one above) should really live in vendor.scss.  Right, @alecmerdler?